.github/workflows/ci.yml

```diff
@@ -36,15 +36,15 @@ jobs:
         id: set_cache
         run: |
           job_cache_extra_deps=(); job_id=test_linux; job_name='Test (Linux)'; job_needs=(); job_os=Linux; job_platform=ubuntu-22.04; job_python=3.9; job_reqs=(reqs/dist.txt reqs/test.txt); job_skip_cache_name=skip_test_linux_py-3.9_ubuntu-22.04; job_skip_cache_path=.skip_cache_test_linux; job_skiplists=(job_test os_linux); job_test_args='-p no:pytest-qt --ignore=test/gui_qt'; job_type=test; job_variant=Linux; analyze_set_job_skip_cache_key
-          job_cache_extra_deps=(osx/deps.sh); job_id=test_macos; job_name='Test (macOS)'; job_needs=(); job_os=macOS; job_platform=macos-10.15; job_python=3.9; job_reqs=(reqs/dist.txt reqs/test.txt); job_skip_cache_name=skip_test_macos_py-3.9_macos-10.15; job_skip_cache_path=.skip_cache_test_macos; job_skiplists=(job_test os_macos); job_test_args='-p no:pytest-qt --ignore=test/gui_qt'; job_type=test; job_variant=macOS; analyze_set_job_skip_cache_key
```

Comment:
You'll get merge conflicts when rebasing/merging master into this branch because I bumped that version. I didn't go for latest though to prevent "random" issues in the future with new macOS versions. In my view freezing this version makes the build more robust. (Same for the other places in this file).

Reply:
Pinning the version is a good idea. I bumped it up to latest because there was some issue with old macOS versions in the CI, and was too lazy to check which versions actually worked. I'll fix the version to match what you had in your patch.